templates/web/fixmystreet-uk-councils/about/googlecookies.html

```diff
@@ -0,0 +1,40 @@
+<h3>Google Analytics Cookies</h3>
```

Comment:
Although there's nothing Merton-specific in this particular file, as it's only used by Merton currently I think it might be better to put it in the `merton` cobrand dir, rather than `fixmystreet-uk-councils`. Easy enough to move it back in the future if another council wants to display the same table.